hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/hevm/HederaEvmTransactionResult.java

```diff
@@ -118,10 +125,12 @@ public ResponseCodeEnum finalStatus() {
         if (haltReason != null) {
             return CustomExceptionalHaltReason.statusFor(haltReason);
         } else if (revertReason != null) {
-            if (revertReason.length() == 0) {
-                return ResponseCodeEnum.CONTRACT_REVERT_EXECUTED;
+            if (revertReason.equals(MAX_STORAGE_EXCEEDED_REASON)) {
+                return MAX_CONTRACT_STORAGE_EXCEEDED;
+            } else if (revertReason.equals(MAX_TOTAL_STORAGE_EXCEEDED_REASON)) {
+                return MAX_STORAGE_IN_PRICE_REGIME_HAS_BEEN_USED;
```

Comment:
Do these just represent the only ones known at this time and this list will be added to as needed?  Or this is it?

Reply:
We will need all 5 from [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/utils/ResponseCodeUtil.java#L60).